src/main/java/Duke/Ui.java

```diff
@@ -0,0 +1,205 @@
+package duke;

```

Comment:
Despite having many different methods in this class, it was still very clear and easy to read because method names were all clear (in camelCase) with javadoc headers for explanations!